source/docs/articles/sites/database.md

```diff
@@ -1,7 +1,18 @@
 ---
 title: MySQL Database
 description: Learn about the Database that runs in your site
-category:
-- developing
-draft: true
+keywords: mysql, sql, database, db, databases
 ---
+The Pantheon platform provides each site a dedicated MySQL container, which can be maintained remotely or locally depending on your workflow. For a comprehensive list of MySQL settings [access your database](/docs/articles/local/accessing-mysql-databases/#database-connection-information) and use the [SHOW VARIABLES](http://dev.mysql.com/doc/refman/5.0/en/show-variables.html) statement.
```

Comment:
@bmackinney can you explain more about "running mariadb" or provide a link you'd like to see included for more info/context? I'm not familiar with it.